src/python/zquantum/core/evolution.py

```diff
@@ -45,7 +45,15 @@ def time_evolution(
 
 
 def _adjust_gate_angle(operation: circuits.GateOperation, time):
-    """Adjust angle in gate operation to account for evolution in time."""
+    """Adjust angle in gate operation to account for evolution in time.
```

Comment:
Actually it looks that this function is no longer used and can be removed.

Reply:
We can also remove single dispatch for `time_evolution_for_term` now, as there's only one implementation :)